### DISCUSSION TITLE
Introduction to Friction2d (Tutorial)

### DISCUSSION THREAD

**Discussion by AlmostMikhail** [friction2d/friction#403] [Show and tell]
![Friction2d.jpg](https://github.com/user-attachments/assets/92826d65-2e59-49b8-aa9c-347271c05ebb)

[Basics of Friction2d](https://youtu.be/XK6x9MWaSrQ?si=KhNNOHUVjv7vVTNL)

**Comment by nuxttux** [comment]
I would like to throw my hat in the ring with this [Ball Bounce Animation - Friction Tutorial](https://youtu.be/F24OzPdf9qc)

![friction_bouncing_ball_Thumbnail_V2](https://github.com/user-attachments/assets/c20ca05b-1e49-42df-9afa-ce8668e64fd4)
